packages/sit-onyx/src/components/OnyxListboxOption/OnyxListboxOption.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <li class="onyx-listbox-option">
+    <span class="onyx-truncation-ellipsis"> {{ $attrs["aria-label"] }}</span>
```

Comment:
I think a slot would be nice here to not rely on the undocumented `aria-label`
```suggestion
    <span class="onyx-truncation-ellipsis"><slot /></span>
```